lib/restore.js

```diff
@@ -29,6 +29,9 @@ var Restore = function(options) {
     d.add(response);
 
     d.on('error', function(error) {
+      if (self._allow.debug) {
+        console.log('error: ', error, error.stack);
```

Comment:
Could you change this to use `process.env.DEBUG` as the flag, please? We already support that option and I don't think we should add a new config option.